acq4/devices/AxoPatch200/AxoPatch200.py

```diff
@@ -126,6 +131,43 @@ def __init__(self, dm, config, name):
         else:
             self.hasSecondaryChannel = False
         
+        if 'AxoPatchVersion' in config :
+			if config['AxoPatchVersion'] == '200A':
+				# telegraph voltage/output translation from the Axopatch 200 amplifier
+				self.mode_tel = np.array([6, 4, 2, 1])
+				self.modeNames = OrderedDict([(0, 'V-Clamp'), (1, 'Track'), (2, 'I-Clamp Normal'), (3, 'I-Clamp Fast'),  ])
+				self.ivModes = {'V-Clamp':'vc', 'Track':'vc', 'I-Clamp Fast':'ic', 'I-Clamp Normal':'ic', 'vc':'vc', 'ic':'ic'}
+				self.modeAliases = {'ic': 'I-Clamp Fast', 'i=0': 'Track', 'vc': 'V-Clamp'}
+				self.requiredSwitchMode = 'Track'
+				self.gain_tel = np.array([0.5,  1.0, 1.5, 2.0, 2.5, 3.0, 3.5, 4.0, 4.5, 5.0, 5.5, 6.0, 6.5])
+				self.gain_vm  = np.array([0.5,  0.5, 0.5, 0.5, 1,   2,   5,   10,  20,  50,  100, 200, 500]) * 1e9  ## values in mv/pA
+				self.gain_im  = np.array([0.05, 0.1, 0.2, 0.5, 1,   2,   5,   10,  20,  50,  100, 200, 500])        ## values in mV/mV
+
+				# Axopatch LPF telegraph
+				self.lpf_tel = np.array([2.0, 4.0, 6.0, 8.0, 10.0])
+				self.lpf_freq = np.array([1.0, 2.0, 5.0, 10.0, 100.0])
+
+			elif config['AxoPatchVersion'] == '200B':
+				# telegraph voltage/output translation from the Axopatch 200 amplifier
+				self.mode_tel = np.array([6, 4, 3, 2, 1])
+				#self.modeNames = OrderedDict([(0, 'V-Clamp'), (2, 'I=0'), (4, 'I-Clamp Fast'), (3, 'I-Clamp Normal'), (1, 'Track'), ])
+				self.ivModes = {'V-Clamp':'vc', 'Track':'vc', 'I=0':'ic', 'I-Clamp Fast':'ic', 'I-Clamp Normal':'ic', 'vc':'vc', 'ic':'ic'}
+				self.modeAliases = {'ic': 'I-Clamp Fast', 'i=0': 'I=0', 'vc': 'V-Clamp'}
+				self.requiredSwitchMode = 'I=0'
+				# Axopatch gain telegraph
```

Comment:
The lpf_tel is the same but the lpf_freq differ between the models. The highest frequency is 50 kHz on the 200A while 100 kHz on the 200B. 
I moved lpf_tel outside the if block.